src/main/java/de/berlin/lostberlin/service/mail/MailService.java

```diff
@@ -28,39 +33,35 @@
         return sender.send(params);
     }
 
-    public Optional<MailSenderResponse> sendNotificationMail(Order order) {
+    public Optional<MailSenderResponse> sendNotificationMail(OrderFullDao order) {
         MailSender sender = MailSenderFactory.getMailSender(MailTypes.NOTIFICATION.name());
         Params params = createNotificationMailParams(order);
         return sender.send(params);
     }
 
     private Params createConfirmationMailParams(Order order) {
-        String subject = "Lost in Berlin Order Confirmation Mail";
-        String toMail = order.getEmail();
+        String subject = "Lost in Berlin order Confirmation Mail";
+        String toMail = order.getEMail();
         Map<String, String> map = new HashMap<>();
         map.put("name", order.getName());
         map.put("orderNumber", order.getOrderNr());
 
         return new Params(fromMail, toMail, subject, map);
     }
 
-    private Params createNotificationMailParams(Order order) {
-        Business business = null;
+    private Params createNotificationMailParams(OrderFullDao order) {
+        Business business;
```

Comment:
You actually do not need this declaration here. It is enough to declare it below when you first get its value.